apps/council/models.py

```diff
@@ -0,0 +1,86 @@
+# -*- encoding: utf-8 -*-
+from django.db import models
+
+TITLES_CHOICES = [
```

Comment:
Choices for charfields should be served as list (or tuple) of tuples.
Details: [choices](https://docs.djangoproject.com/en/1.8/ref/models/fields/#choices)